$OpenBSD: patch-content_child_child_process_sandbox_support_impl_linux_cc,v 1.1 2019/11/10 13:37:53 robert Exp $

commit 2b580e805103141bb1d9da1e120db17911a14c49
Author: Dominik R_ttsches <drott@chromium.org>
Date:   Mon Sep 30 19:42:18 2019 +0000

    Remove noisy error log for failing to find font fallback

Index: content/child/child_process_sandbox_support_impl_linux.cc
--- content/child/child_process_sandbox_support_impl_linux.cc.orig
+++ content/child/child_process_sandbox_support_impl_linux.cc
@@ -47,10 +47,8 @@ void WebSandboxSupportLinux::GetFallbackFontForCharact
   std::string family_name;
   if (!font_loader_->FallbackFontForCharacter(character, preferred_locale,
                                               &font_identity, &family_name,
-                                              &is_bold, &is_italic)) {
-    LOG(ERROR) << "FontService fallback request does not receive a response.";
+                                              &is_bold, &is_italic))
     return;
-  }
 
   // TODO(drott): Perhaps take OutOfProcessFont out of the picture here and pass
   // mojo FontIdentityPtr directly?
